### ISSUE TITLE
Running in as systemd daemon causes high CPU usage

### ISSUE THREAD

**Issue by SISheogorath** [SFTtech/sftmumblebot#22]
I run sftmumblebot inside a docker container. And it themes like detaching the container from stdin causes high CPU load. 

As it started flooding the IRC and the mumble channel instantly it joined I "fixed" it by using the following patch: https://github.com/SISheogorath/docker-sftmumblebot/blob/b05ec9280fe62c799595d762d57f694e864b04a9/console-fix.patch

But I reconized that didn't lower the CPU usage. So here is my new patch: https://github.com/SISheogorath/docker-sftmumblebot/commit/60199b6c7dd50fe95b1ec71be86432eccd6e7bc9

This solves the CPU problem but its a bad solution. May fix it in general?

**Comment by TheJJ** [comment]
Yes, you can fix it in general, i'd recommend having a flag that enables or disables the console feedback. We can also add the systemd files to `dist/` if you like :)
